### ISSUE TITLE
Fine-tune recipe/guidelines for adapting an existing adapter/model to more languages/tasks with SFT/DPO.

### ISSUE THREAD

**Issue by samuelazran** [fixie-ai/ultravox#168]
First, thanks for publishing this incredible work.

I'm interested in testing this model for ASR in languages not in the adapter training data and looking for examples/guidelines on fine-tuning an existing adapter. Is it better to train an adapter from scratch on my multi-lingual datasets? what are good values for learning rates/batch size, etc?

Also, I am looking for any recommendations or experience with post-training or DPO on a voice model like this one.

**Comment by zqhuang211** [comment]
We recommend starting with our pretrained model,  whether you're working on general-purpose tasks (using the continuation training data) or focusing on a specific task like ASR. 

To get started, you can follow the instruction in https://github.com/fixie-ai/ultravox/?tab=readme-ov-file#training, and set the `model_load_dir` parameter to the pretrained model (see: https://github.com/fixie-ai/ultravox/blob/87acdb8dfff8287985eff12a02e74e0c2df21caf/ultravox/training/config_base.py#L75C5-L75C19) 

If you're only interested in ASR and not other capabilities, we suggest setting `loss_function` to `CrossEntropy` in the experiment config file (e.g., https://github.com/fixie-ai/ultravox/blob/main/ultravox/training/configs/release_config.yaml). This fine-tunes the model exclusively for the ASR task, though it will lose its general instruction-following capabilities. Otherwise, we recommend keeping the default KL_Divergence loss, which retains the model’s versatility across tasks.
